services/workflows-service/prisma/migrations/20240522121355_alert_dev_cross_env_not_null/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Made the column `crossEnvKey` on table `AlertDefinition` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "AlertDefinition" ALTER COLUMN "crossEnvKey" SET NOT NULL;
```

Comment:
Have you checked in all env that we do have crossEnvKey for all alert definitions ?

Reply:
We did it together in the office afair